io.catenax.shared.shopfloor_information_types/1.0.0/metadata.json

```diff
@@ -1 +1 @@
-{ "status" : "release" }
+{ "status" : "deprecated" }
```

Comment:
ok, then i have a question, what should be set for the metadata.json. I expected that i should be deprecate, since we are publishing version 2.0.0. However, if possible, keep version 1.0.0. SO what do i have to then write into the status?

Reply:
then you don't need to change anything in status. you can keep it `release` like before and revert the changes. Let's deprecate it later on.

Reply:
Thank you for your fast replay, i reverted it to release and pushed the latest version